src/wp-includes/class-wp-scripts.php

```diff
@@ -294,15 +294,42 @@ public function do_item( $handle, $group = false ) {
 			$cond_after  = "<![endif]-->\n";
 		}
 
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+
 		$before_handle = $this->print_inline_script( $handle, 'before', false );
-		$after_handle  = $this->print_inline_script( $handle, 'after', false );
 
 		if ( $before_handle ) {
 			$before_handle = sprintf( "<script%s id='%s-js-before'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $before_handle );
 		}
 
-		if ( $after_handle ) {
-			$after_handle = sprintf( "<script%s id='%s-js-after'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $after_handle );
+		$after_handle = '';
+		if ( '' === $strategy ) {
```

Comment:
It may be worth an inline comment here?
```suggestion
		// Eligible loading strategies will only be 'async', 'defer', or ''.
		if ( '' === $strategy ) {
```

Reply:
Done.